code/modules/modular_computers/file_system/programs/jobmanagement.dm

```diff
@@ -21,7 +21,6 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		JOB_HEAD_OF_SECURITY,
 		JOB_RESEARCH_DIRECTOR,
 		JOB_SECURITY_MEDIC, // SKYRAT EDIT ADD START
-		JOB_CLOWN,
```

Comment:
😕

Reply:
You do realize this is just a revert of a skyrat edit & allows the hop to add more open clown slots? This seems like a good change to me.